toolkit/src/components/cut/list-item/action-generic.hbs

```diff
@@ -0,0 +1,5 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+{{yield to='div'}}
```

Comment:
looking at the code, I am not sure you need the `to='div'` to work, probably you can just remove it
```suggestion
{{yield}}
```